experimental/logging/log_listener.go

```diff
@@ -91,7 +90,7 @@ func (f *loggingListenerFactory) NewListener(fnd api.FunctionDefinition) experim
 			return nil // not yet supported
 		}
 	default:
-		if f.fsOnly {
+		if f.scopes.IsInLogScope(logging.LogScopeFilesystem) {
```

Comment:
this is breaking my brain at the moment.. not sure it fits to say anything about FS anymore

Reply:
yeah I changed that to `f.scopes!=0` (and actually `f.scopes.Defined()`)